core/src/scss/components/card/_card--minimal.scss

```diff
@@ -0,0 +1,20 @@
+.su-card--minimal {
+  @include box-shadow($depth: 'flat', $stroke: 'none');
+  background-color: inherit;
+
+  .su-card__contents {
+    // override modular spacing at all breakpoints
+    @include grid-media-only(all) {
+      padding-left: 0;
+      padding-right: 0;
```

Comment:
For the vertical card, I understand keeping the padding-top so there's space between the image and the content, but we probably should set `padding-bottom: 0` also, since we don't add any padding/margin on the "outside" of any components in Decanter.

Reply:
done

Reply:
Actually could we use the padding mixin here
`@include padding(null 0 0);`
Sorry I forgot about that